openshift/buildConfig-template.yaml

```diff
@@ -139,7 +433,9 @@ objects:
           RUN \
             useradd -m solver && \
             dnf update --setopt=tsflags=nodocs -y && \
-            dnf install --setopt=tsflags=nodocs -y python36 python3-pip && \
+            dnf install --setopt=tsflags=nodocs -y python36 wget && \
+            wget https://bootstrap.pypa.io/get-pip.py && \
+            /usr/bin/python3.6 get-pip.py &&\
```

Comment:
There is no pip available via rpm?

Reply:
For Fedora the rpm's it was py3.7 https://fedora.pkgs.org/30/fedora-x86_64/python3-pip-19.0.3-1.fc30.noarch.rpm.html
Is there an efficient way to look for RPMs?

Reply:
Indeed, there is just one pip for the default python interpreter packaged shipped. :+1: